### DISCUSSION TITLE
Shader Editor

### DISCUSSION THREAD

**Discussion by mingle47** [nem0/LumixEngine#1382] [General]
I saw in a youtube video(2019) that there was a shader editor. 
![LumixShaderEditor](https://user-images.githubusercontent.com/5350158/116222459-f821e700-a791-11eb-894c-6225d7f12a08.jpg)
This does not seem to be in the current version.
I downloaded the source and compiled and still seems not to be available. Has this been removed?
Thanks

**Comment by nem0** [comment]
It's a plugin, see https://github.com/nem0/LumixEngine/wiki/available-plugins for more information how to get it. I must warn you though, it's not used by anybody and while it's possible to make some basic stuff with it, it's not polished and a lot of functionality is not exposed at all.
